libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -146,17 +152,19 @@ export default function TeamCharacterSelector({
             }}
           >
             <Typography
```

Comment:
I think we can simplify the hierarchy:
-CardContent (flex, gap, align+justify:center, hover)
  - TeamIcon
  - Typography ( nowrap, h5, textshadow)

Reply:
this worked 👍